### PR TITLE
Update ie_resources.md

### PR DESCRIPTION
**Pull request by clearysi**
Updated Institutional resources link for UCD.
Added more detail to the intro

### CODE REVIEW

**Review by Nazeeefa — APPROVED** [review]

File: pages/national_resources/ie_resources.md
Comment (bedroesb) [inline pages/national_resources/ie_resources.md:1]: Are there data management tools/resources that you would like to highlight? You can list them under `national_resources` like is done on the other country pages.

Comment (clearysi) [reply pages/national_resources/ie_resources.md:1]: Currently, we don't have a national data management tool to highlight in this section. If one becomes available I will add it to the page. Thank you!

Comment (bedroesb) [reply pages/national_resources/ie_resources.md:1]: ok no problem!

**Review by bedroesb — APPROVED** [review]

**Comment by bedroesb** [comment]
When you are happy with the content and think that the page is ready to go live (visible from the navigation structures), please let us know here: https://github.com/elixir-europe/rdmkit/pull/1271 .

**Comment by Nazeeefa** [comment]
@clearysi for future updates to the page, I was wondering if you are know of Irish Data Stewardship Network “Sonraí“? https://datastewards.ie/, apparently soon to be launched. Could be worth mentioning :)

**Comment by clearysi** [comment]
Hi @Nazeeefa, thank you! Yes, I will add it when the website is live.

**Pull request merged**